### PR TITLE
chore: Comment 3.8 PY required CI checks

### PR DESCRIPTION
**Pull request by EugeneTorap**
### SUMMARY
Before remove 3.8 PY support and bump to 3.9 PY version in #23890 we need to comment 3.8 PY CI tests in github config file.

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Review by villebro — COMMENTED** [review]
While I agree that this is the end state we want to achieve, this change will block all other PRs until the CI bump PR is merged. So if we run into trouble with it we're screwed and may need ASF help to unblock us. Let's rather remove the required 3.8 tests, leave the 3.9 ones and reintroduce the 3.8 and 3.9 ones as 3.9 and 3.10 respectively after the CI update has been merged, just to be safe.

**Review by villebro — APPROVED** [review]
LGTM - Let's try to follow up quickly with the next step so we can re-enable the bumped workflows

**Review by dpgaspar — APPROVED** [review]

**Pull request merged**
